src/pages-and-resources/discussions/DiscussionsSettings.jsx

```diff
@@ -114,7 +122,17 @@ function DiscussionsSettings({ courseId, intl }) {
               eventKey={SELECTION_STEP}
               title={intl.formatMessage(messages.providerSelection)}
             >
-              <AppList />
+              {courseDetail
+                && !canChangeProviders
+                && (
+                  <Alert variant="warning">
+                    <FormattedMessage
+                      id="authoring.discussions.noProviderSwitchAfterCourseStarted"
+                      defaultMessage="Discussion provider can't be changed after course has started, please reach out to partner support."
+                    />
```

Comment:
Move this to the messages file.

Reply:
Hmm, any example of what you mean? All the instances of `<FormattedMessage>` I've found are done like this.

Reply:
Check out the `messages.js` file and how `intl.formatMessage` is used. What you've done is perfectly valid, however since the other strings in this file are loaded from messages.js this should as well to be consistent.